src/Akka.Persistence.Redis.Tests/Query/RedisCurrentPersistenceIdsSpec.cs

```diff
@@ -39,12 +39,33 @@ public RedisCurrentPersistenceIdsSpec(ITestOutputHelper output, RedisFixture fix
         {
             ReadJournal = Sys.ReadJournalFor<RedisReadJournal>(RedisReadJournal.Identifier);
         }
-
-        [Fact(Skip = "Not implemented yet")]
-        public override void ReadJournal_query_CurrentPersistenceIds_should_not_see_new_events_after_complete()
+        [Fact]
+        public void ReadJournal_CurrentPersistenceIds_should_fail_the_stage_on_connection_error()
```

Comment:
It has been superseded by this
https://github.com/eaba/Akka.Persistence.Redis/blob/208d831d168aafb44c756153db8e8144914e6f89/src/Akka.Persistence.Redis.Tests/Query/RedisCurrentPersistenceIdsSpec.cs#L42-L68

Reply:
A test must perform a test, it is useless to have an empty test that does nothing. A test method name indicates that a certain test case has been tested and the result shows whether there was a success or failure to the test specification.
By adding a `FactAttribute` to a test method, the method is collected by the XUnit framework and run during the test session. 
What you're saying, in code, is that ReadJournal CurrentPersistenceIds __will__ fail the stage when a connection error occured, but you provide __zero proof__ that it will, because the method body is empty.
A person who doesn't have time to read the code and only have a test result to go by will be __fooled__ because he will see that the ReadJournal is failing the stage, but it might not, because your code __lied__, it didn't test __anything__.

Reply:
I understand - that is now outdated.